spec/system/candidate_interface/signup_and_signin/candidate_signs_up_using_magic_link_with_an_invalid_token_spec.rb

```diff
@@ -54,7 +53,7 @@ def then_i_receive_an_email_inviting_me_to_sign_up
   end
 
   def when_the_magic_link_token_is_overwritten
-    Candidate.find_by(email_address: @email).update(magic_link_token: MagicLinkToken.new.raw)
+    Candidate.find_by(email_address: @email).authentication_tokens.delete_all
```

Comment:
Do we need a system spec for signing in with old (but valid) magic links?

Reply:
Have added one 👍